packages/binding-opcua/README.md

```diff
@@ -0,0 +1,81 @@
+ 
+* URI schema: the schema for OPC UA is not registered with IANA; the Binding is using `opc.tcp`.
+* IP address and port: IP address and port of the OPC UA server. The credentials for connecting to the server can be added into the `wot-servient.configuration.json`.
+* NodeID: the NodeID of the node addressed by the given InteractionAffordance. The NodeID follows the [XML notation](https://documentation.unified-automation.com/uasdkhp/1.0.0/html/_l2_ua_node_ids.html), like several Graphical tools for exploring OPC UA devices. The NodeID is composed by the namespace index, the identifier type, and the node Identifier. If a method has to be addressed, since it requires to be attached to a device, both the DeviceId and the MethodId must be included to the NodeID. In particular, for convenience, the DeviceId comes before the MethodId. 
+
+This is a valid `href` example for addressing a node:
+`opc.tcp://localhost:5050/ns=1;s=mynode`
+
+For a method, two nodes need to be given (one for the node on which to call the method, one for the method definition):
+`opc.tcp://localhost:5050/ns=1;s=mydevice;mns=1;mb=9997FFAA`
+
+where `ns=1;s=mydevice` is the nodeId of the Device, while `mns=1;mb=9997FFAA` is the nodeId of the method to apply.
+ 
+### opc:method
+ 
+The optional attribute `opc:method` specifies which kind of call should be used in the request.
+For the state of this implementation, it is still not used.
+These are the default values for this Binding:
+
+* readproperty is set to `READ`
+* writepropery is set to `WRITE`
+* invokeaction is set to `CALL_METHOD`
+ 
+## New DataSchema Fields for the OPC UA Binding
+ 
+OPC UA uses custom Datatypes on the wire, which requires additional translations from the JSON model used at WoT application level.
+To enable support for a proper Datatype translation, the DataSchema information has to be extended with the following terms, and the schema also be passed down to the Binding.
+ 
+### opc:DataType
+
+Among all the possible OPC UA Datatypes, at the moment all the ones supported by (node-wot)[https://github.com/node-opcua/node-opcua/blob/master/packages/node-opcua-variant/source/DataType_enum.ts] can be used. 
+The binding uses a new custom ContentSerdes in order to handle the proper OPC UA Datatype and adjust the OPC UA request accordingly. Please note that this binding does not directly handle OPC UA raw bytes but instead an intermediate format to be passed to another NodeJS library ([node-opcua](https://github.com/node-opcua)) that is in charge of making the raw conversion. In this sense, the new ContentSerdes strongly depends on the Nodejs library used behind.
+
+## Additional tools
+
+A basic OPC UA server crawler and a basic OPC UA -> TD translator can be found [here](https://github.com/lukesmolo/wot-utils/tree/master/opcua-crawler)
+
+## TODO
+ 
+- [ ] Subscriptions implementation (EVENTS) with Sub/Pub protocol
+- [x] TEST
+- [ ] (OPC UA Server Protocol Binding ?)
+ 
+## Acknowledgments
```

Comment:
Please remove any acknowledgments. We do not have it elsewhere in the project.

Reply:
ok, done